src/components/Dialog/Dialog.tsx

```diff
@@ -12,11 +12,12 @@ type Props = {
   onClose: () => void;
   size?: 'small' | 'large';
   children: React.ReactNode;
+  role?: string;
```

Comment:
Just to make it a tiny bit better ;-)

```suggestion
  role?: role: React.AriaRole;
```